nasa-gibs/worldview
# Add two tempo layers to EIC

## Description

> The Earth Information Center would like the include TEMPO products on the EIC.

> The TEMPO data products we would like to bring to the AQ EIC dashboard are:

> Nitrogen Dioxide (L3, Vertical Column Troposphere)
> Ozone (L3, Column Amount O3, Subdaily)


## How To Test

1. `git checkout WV-3199-TEMPO-EIC`
2. `npm run watch`
3. Open this [link](http://localhost:3000/?v=-158.20387917178206,-4.6433540592384475,-19.848061739477203,74.86320074995064&df=true&kiosk=true&eic=si&scenario=14&l=Coastlines_15m,TEMPO_L3_NO2_Vertical_Column_Troposphere&lg=false)
4. After a second or two the date should jump back to June 3rd 2024 and imagery should load.
5. Open this [link](http://localhost:3000/?v=-170.45859053523907,-12.106744252573016,-12.991540899766122,78.3821332289227&df=true&kiosk=true&eic=si&scenario=15&l=Coastlines_15m,TEMPO_L3_Ozone_Column_Amount&lg=false)
6. After a second or two the date should jump back to June 3rd 2024 and imagery should load.

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
